css/preflight.css

```diff
@@ -508,6 +508,11 @@ fieldset {
   padding: 0;
 }
 
+ol,
+ul {
+  margin: 0;
```

Comment:
Would it make more sense to move this to our basecss fork?

https://github.com/tailwindcss/tailwindcss/blob/327e6d3cab06dcfab58866178a6cb041383f194a/css/preflight.css#L479-L489

Reply:
I considered that but those selectors are in alphabetical order, and keeping things in alphabetical order would separate ol and ul from each other 🤔  So opted to just keep it as an explicit rule, although I'm also happy to move it there if you think that's better.

Reply:
Nah I don't care that much. My only thinking was trying to keep normalize as standard as possible, and then only making tweaks to our basecss fork.

But maybe we're already making other (non-comment) changes to normalize?

Reply:
It **is** in the SUIT CSS Base fork, not in Normalize. I’m confused?